.github/test-payload.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ref": "release/v55.10.2022082401"
```

Comment:
This is used as a part of testing with Act. I still need to add some doc on how to invoke this!

Reply:
👍 should the file be committed or added to .gitignore since it's only used for local testing?

Reply:
I think removing altogether might be the call here, and I'll just update my local .gitignore